recipes-kernel/linux/4.19/linux-openxt_4.19.20.bb

```diff
@@ -0,0 +1,53 @@
+DESCRIPTION = "Linux kernel for OpenXT service VMs."
+
+# Use the one from meta-openembedded/meta-oe
+require recipes-kernel/linux/linux.inc
+require recipes-kernel/linux/linux-openxt.inc
+
+PV_MAJOR = "${@"${PV}".split('.', 3)[0]}"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/patches:${THISDIR}/defconfigs:"
+SRC_URI += "${KERNELORG_MIRROR}/linux/kernel/v${PV_MAJOR}.x/linux-${PV}.tar.xz;name=kernel \
+    file://bridge-carrier-follow-prio0.patch;patch=1 \
```

Comment:
All these `patch=1`  flags can be dropped since bitbake treats .patch files as patches by default.

Reply:
Agreed, fix coming.